.github/workflows/02_scala_container_publishing.yml

```diff
@@ -1,9 +1,16 @@
-name: Publish Scala Container Images
+name: 02. Publish Scala Container Images
 
 on:
   push:
-    # Publish semver tags as releases.
-    tags: [ 'v*.*.*' ]
+    tags:
+      - 'v*.*.*'  # Trigger on semver tag release (v1.2.3)
+  workflow_run:
+    workflows:
+      - "01. Scala Build & Test"
```

Comment:
Should this run when a PR is created but _before_ that PR is merged? I think that's how this will end up.